twilio/http/http_client.py

```diff
@@ -13,7 +14,7 @@ class TwilioHttpClient(HttpClient):
     """
     General purpose HTTP Client for interacting with the Twilio API
     """
-    def __init__(self, pool_connections=True, request_hooks=None, timeout=None, logger=_logger, proxy=None):
+    def __init__(self, pool_connections=True, request_hooks=None, timeout=None, logger=_logger, proxy=None, max_retries=3):
```

Comment:
Prefer to default this to `None` (and adding the proper `None`-checking logic) so as not to impact other users.

Reply:
@childish-sambino Done: https://github.com/twilio/twilio-python/pull/507/commits/ef86ddbad322ba79779161151063b133e2e77a36

Reply:
Could you please take another look?